server/src/service/machinesSchema.ts

```diff
@@ -0,0 +1,75 @@
+import { gql } from "apollo-server-express";
+
+export const MachinesTypeDefs = gql`
+  type Machine {
+    id: ID!
+    machineFamily: Int!
+    name: String!
+    room: String!
+    addedAt: Date
+    inUse: Boolean!
+  }
+
+  type MachineFamily {
+    id: ID!
+    name: String!
+    description: String
+    trainingModule: Int
+  }
+
+  type Reservation {
+    id: ID!
+    studentId: Int!
+    machineId: Int!
```

Comment:
same thing as above. These Id fields really should just be actual type instead of an int. That way graphql can resolve everything the client asks for in a single request